packages/node/src/methods/state-channel/create/controller.ts

```diff
@@ -50,39 +31,28 @@ export default class CreateChannelController extends NodeController {
     requestHandler: RequestHandler,
     params: Node.CreateChannelParams
   ): Promise<Node.CreateChannelTransactionResult> {
-    const { owners, retryCount } = params;
-    const { wallet, networkContext, provider, store } = requestHandler;
+    const { owners } = params;
+    const { networkContext, store } = requestHandler;
 
     const multisigAddress = getCreate2MultisigAddress(
       owners,
       networkContext.ProxyFactory,
       networkContext.MinimumViableMultisig
     );
 
-    // By default, if the contract has been deployed and
-    // DB has records of it, controller will return HashZero
-    let tx = { hash: HashZero } as TransactionResponse;
+    // always will be HashZero since we are deploying on withdrawal
```

Comment:
Alternatively we can update the API for this method.

IMO I don't think backward compatibility is a concern atm.

Reply:
API updated, this line is deleted.